google-cloud-tools-plugin/src/com/google/cloud/tools/intellij/appengine/facet/AppEngineStandardLibraryPanel.java

```diff
@@ -65,67 +60,47 @@ public AppEngineStandardLibraryPanel(boolean enabled) {
     // Objectify and Endpoints are dependencies of the App Engine API.
     objectifyCheckBox.setSelected(false);
     endpointsCheckBox.setSelected(false);
-    appEngineApiCheckBox.addItemListener(new ItemListener() {
-      @Override
-      public void itemStateChanged(ItemEvent event) {
-        JCheckBox checkbox = (JCheckBox) event.getItem();
-
-        if (!checkbox.isSelected()) {
-          objectifyCheckBox.setSelected(false);
-          endpointsCheckBox.setSelected(false);
-        }
+    appEngineApiCheckBox.addItemListener(event -> {
+      JCheckBox checkbox = (JCheckBox) event.getItem();
+
+      if (!checkbox.isSelected()) {
+        objectifyCheckBox.setSelected(false);
+        endpointsCheckBox.setSelected(false);
       }
     });
-    objectifyCheckBox.addItemListener(new AppEngineApiDependencyCheckboxListener());
-    endpointsCheckBox.addItemListener(new AppEngineApiDependencyCheckboxListener());
+    objectifyCheckBox.addItemListener(this::listenAppEngineApiDependencyCheckbox);
```

Comment:
I like the refactoring, but not the name choice for the method. Maybe handleAppEngineApiDependencyCheckboxEvent?